.github/ISSUE_TEMPLATE/desktop--bug-report.md

```
@@ -0,0 +1,86 @@
+Linux (deb/rpm):
+
+```sh
+desktop
+# or, if not on PATH:
+/opt/Grida/desktop
+```
+
+<details>
+<summary>Terminal output</summary>
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)